src/components/NurseryWithdrawals/WithdrawalDetails/sections/NonOutplantWithdrawalTable.tsx

```diff
@@ -34,6 +36,14 @@ const columns = (): TableColumnType[] => [
   { key: 'total', name: strings.TOTAL, type: 'number' },
 ];
 
+const nurseryV2Columns = (): TableColumnType[] => [
```

Comment:
should this have a germinating quantity column?

Reply:
I missed it on designs! Looks like for "Nursery Transfer" is not there, but it is for "Other/dead". But I saw you added the input for Nursery Transfer too. Should I add it to all of them?

Reply:
I just x-referenced the PRD. outplant doesn't have germinating, all other purposes do. Tagged you in a comment in prd

Reply:
Thanks!